tests/test_material.py

```diff
@@ -46,6 +47,11 @@
 
 leu = {922380000: 0.96, 922350000: 0.04}
 
+# Relative Tolerance
+rtol = 1e-10
```

Comment:
Do we know if this works at 1e-12?

Reply:
We can test this while working on PR #1549. For now I am setting the value to 1e-20.